src/App.js

```diff
@@ -14,11 +16,34 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    goods: goodsFromServer,
+    showGoods: false,
+  }
+
+  showGoods = () => {
+    this.setState(state => ({
+      showGoods: !state.showGoods,
+    }));
+  }
+
+  render() {
+    return (
+      <div className="App">
+        <button
+          type="button"
+          onClick={this.showGoods}
+        >
+          Start
+        </button>
+
+        {this.state.showGoods && (
+        <GoodsList goods={this.state.goods} />
```

Comment:
```suggestion
        <GoodsList goods={goodsFromServer} />
```
You can get rid from this value in the state and pass goodsFromServer directly